app.rb

```diff
@@ -0,0 +1,172 @@
+require './book'
+require './teacher'
+require './student'
+require './rental'
+
+class App
+  attr_reader :books, :people, :rental
+
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  def run
+    show_menu
+
+    menu_list = {
+      '1' => method(:book_list),
+      '2' => method(:people_list),
+      '3' => method(:add_teacher),
+      '4' => method(:add_student),
```

Comment:
- [ ] It will be great to check if the student has parental permission when creating a new student. At the moment, this option is not showing. Kindly fix this implementation